api/src/main/java/org/ccci/idm/user/dao/ldap/Constants.java

```diff
@@ -21,6 +21,8 @@
     public static final String LDAP_ATTR_PASSWORDCHANGEDTIME = "pwdChangedTime";
     public static final String LDAP_ATTR_FACEBOOKID = "thekeyFacebookId";
     public static final String LDAP_ATTR_FACEBOOKIDSTRENGTH = "thekeyFacebookIdStrength";
+    public static final String LDAP_ATTR_GRPERSONID = "thekeyGrPersonId";
```

Comment:
this is internal for eDirectory only, I want to keep our custom attributes prefixed with something common for organization.